Cargo.toml

```diff
@@ -3,7 +3,11 @@ name = "pwasm-test"
 version = "0.1.0"
 authors = ["NikVolf <nikvolf@gmail.com>", "Alexey Frolov <frol.rage@gmail.com>"]
 
+[dependencies.pwasm-std]
+version = "0.3"
+features = ["std"]
```

Comment:
why std? was ok without, wan't it?

Reply:
The idea was to hardcode std to deps. And rid of explicit set of features
 https://github.com/paritytech/pwasm-test/pull/22/commits/477ee8612a460461e6bc7011f2af40ab763a5e37